code/modules/research/xenobiology/xenobiology.dm

```diff
@@ -256,6 +263,58 @@
 			qdel(src)*/
 
 
+/obj/item/weapon/slimestabilizer
+	name = "slime stabilizer"
+	desc = "A potent chemical mix that will reduce the chance of a slime mutating."
+	icon = 'icons/obj/chemical.dmi'
+	icon_state = "bottle15"
+
+/obj/item/weapon/slimestabilizer/attack(mob/living/simple_animal/slime/M as mob, mob/user as mob)
+	if(!isslime(M))
+		user << "<span class='warning'> The stabilizer only works on baby slimes!</span>"
+		return ..()
+	if(M.stat)
+		user << "<span class='warning'> The slime is dead!</span>"
+		return..()
+	if(M.mutation_chance == 0)
+		user <<"<span class='warning'> The slime already has no chance of mutating!</span>"
+		return..()
+
+	user <<"<span class='notice'>You feed the slime the stabilizer. It is now less likely to mutate.</span>"
+	M.mutation_chance -= 15
```

Comment:
Oh yeah, I meant to include the -15 in the clamp statement.

```Clamp(value, minimum, maximum)```
it puts the number ```value``` between ```minimum``` and ```maximum```
So if you range includes negatives (Clamp(health, -100, 100)) then it handles them.